### DISCUSSION TITLE
VSCode is warning `import type { AppProps } from "next/app"`

### DISCUSSION THREAD

**Discussion by gr-qft** [vercel/next.js#17326] [Help] [answered]
Inside `_app` I'm writing `import type { AppProps } from "next/app"` (from Next.js doc) but VSCode is giving me a warning. If I write  `import { AppProps } from "next/app"` instead everything would be fine. Which one is the right option?

**Comment by lfades** [comment, marked as answer]
[`import type`](https://www.typescriptlang.org/docs/handbook/modules.html#importing-types) is always guaranteed to be removed from your bundle, although normal imports will also be removed if you only import types, so there should not be a noticeable difference.

VS Code may be warning you if you're on an out of date TS version, as `import type` is only available since TS 3.8

**Reply by gr-qft** [reply]
I see, thanks for the info! Btw, why did you say **noticeable**, shouldn't there be no difference at all?

**Reply by lfades** [reply]
Yeah, There shouldn't be any difference at all.

**Reply by gr-qft** [reply]
👍
